backend/src/kitconcept/core/behaviors/image.py

```diff
@@ -0,0 +1,17 @@
+from plone.autoform.interfaces import IFormFieldProvider
+from plone.namedfile import field as namedfile
+from plone.supermodel import model
+from plone.volto import _
+from zope.interface import provider
+
+
+@provider(IFormFieldProvider)
+class IImage(model.Schema):
+    image = namedfile.NamedBlobImage(
+        title=_("label_image", default="Image"),
+        description=_(
+            "help_image",
+            default="Insert an image that will be used in listing and teaser blocks.",
```

Comment:
We'll also show it on the person profile view, right? Probably we want a behavior with text specifically for the person content type. FZJ has the description "Use a normal portrait image with your face centered" which seems helpful.

Reply:
Yeah, https://gitlab.kitconcept.io/kitconcept/distribution-kitconcept-intranet/-/issues/133#note_21937